packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2906,6 +2911,11 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   ///
   /// The `duration` parameter can be set to a non-zero value to bring the
   /// target object on screen in an animation defined by `curve`.
+  ///
+  /// See also:
+  ///
+  /// * [RenderViewportBase.showInViewport], which this method typically
+  /// delegates to.
```

Comment:
indent

Reply:
It maybe a little bit inaccurate, we do not override this method often, it just so happens that we uses this method for those occasion.